config/default.yaml

```diff
@@ -2,8 +2,8 @@
 slack:
   #bot_token: no default
   #app_token: no default
-  report_channel: bot-test
```

Comment:
given we need the ID for filtering message AND the API supports using IDs instead of name, lets drop this configuration option as its unneeded because we can use the ID instead.